src/sage/graphs/generators/families.py

```diff
@@ -3920,7 +3920,11 @@ def MathonPseudocyclicStronglyRegularGraph(t, G=None, L=None):
         sage: G.is_strongly_regular(parameters=True)                                    # needs sage.modules sage.rings.finite_rings
         (45, 22, 10, 11)
 
-    Supplying ``G`` and ``L`` (constructed from the automorphism group of ``G``). ::
+    Supplying ``G`` and ``L`` (constructed from the automorphism group
+    of ``G``). The entries of L can't be tested directly because
+    there's some unpredictability in the way that GAP chooses a
+    representative in ``NormalSubgroups()``, the function that
+    underlies our own ``normal_subgroups()`` method::
```

Comment:
`:meth:`?

Reply:
These cross-class method references always give me headaches but let's see how it turns out. Done (maybe).

Reply:
Trying again with the missing tilde.